lib/src/transaction_builder/transfer_transaction_builder.ts

```diff
@@ -0,0 +1,334 @@
+import ow from 'ow';
```

Comment:
comment about what "ow" means would be good

Reply:
`ow` is a library for doing runtime types check in JS. I tried many of them and it is the best I found. I am not sure whether we want to document a dependency is. Do u have any idea?

Reply:
i see, i think ow is good

Reply:
I think it's unnecessary for us to document `ow`, since it's transparent to the users(developers) and `ow` is just checking the types behind and complaining about the error to them. Btw, I think `ow` is a great choice, and the author is a renowned open source contributor.